### ISSUE TITLE
Get-MgEntitlementManagementAccessPackageAssignmentPolicy - No HTTP resource was found

### ISSUE THREAD

**Issue by petr-stupka** [microsoftgraph/msgraph-sdk-powershell#786]
```powershell
Get-MgEntitlementManagementAccessPackageAssignmentPolicy -AccessPackageId 'ed8b14c2-62f0-4b40-a67a-7d362260119d'
```
**Error:**

`
No HTTP resource was found that matches the request URI 'https://elm.iga.azure.com/api/v1/accessPackages('ed8b14c2-62f0-4b40-a67a-7d362260119d')/accessPackageAssignmentPolicies'.
`

**Debug:**

```powershell
DEBUG: CmdletBeginProcessing: 
DEBUG: CmdletProcessRecordStart: 
DEBUG: CmdletProcessRecordAsyncStart:
DEBUG: CmdletGetPipeline:
DEBUG: CmdletBeforeAPICall:
DEBUG: URLCreated: https://graph.microsoft.com/beta/identityGovernance/entitlementManagement/accessPackages/ed8b14c2-62f0-4b40-a67a-7d362260119d/accessPackageAssignmentPolicies
DEBUG: RequestCreated: https://graph.microsoft.com/beta/identityGovernance/entitlementManagement/accessPackages/ed8b14c2-62f0-4b40-a67a-7d362260119d/accessPackageAssignmentPolicies
DEBUG: HeaderParametersAdded: https://graph.microsoft.com/beta/identityGovernance/entitlementManagement/accessPackages/ed8b14c2-62f0-4b40-a67a-7d362260119d/accessPackageAssignmentPolicies
DEBUG: GET /beta/identityGovernance/entitlementManagement/accessPackages/ed8b14c2-62f0-4b40-a67a-7d362260119d/accessPackageAssignmentPolicies HTTP/1.1
HTTP: graph.microsoft.com
```

**Possible solution:**

add query `?expand=accessPackageAssignmentPolicies` instead of path `/accessPackageAssignmentPolicies`

PSVersion - 7.1.3
Module version - 1.6.1 
[AB#10420](https://microsoftgraph.visualstudio.com/0985d294-5762-4bc2-a565-161ef349ca3e/_workitems/edit/10420)

**Comment by peombwa** [comment]
Thanks for surfacing this. This bug has already been fixed by https://github.com/microsoftgraph/msgraph-sdk-powershell/pull/723 and will be released later this in 1.7.0.

**Comment by jebalert** [comment]
Re-opening. This issue is present in 2.27
